src/programQueueProcessor/unit.test.ts

```diff
@@ -251,6 +255,44 @@ describe("kafka integration", () => {
       ).body?.hits?.total?.value;
       expect(test_ca_documents).to.equal(testDonorIds.length);
 
+      // check if new rdpc data is relfected in TEST-CA
+      testDonorIds.forEach(async (donorId) => {
+        const esQuery = esb
+          .requestBodySearch()
+          .size(testDonorIds.length)
+          .query(esb.termQuery("donorId", donorId));
+
+        const test_ca_hits: EsHit[] = await esClient
+          .search({
+            index: ALIAS_NAME,
+            body: esQuery,
+          })
+          .then((res) => res.body.hits.hits)
+          .catch((err) => {
+            return [];
+          });
+
+        expect(test_ca_hits.length).to.equal(1);
+        expect(test_ca_hits[0]._source.alignmentsCompleted).to.equal(
+          expectedRDPCData[donorId].alignmentsCompleted
+        );
+        expect(test_ca_hits[0]._source.alignmentsFailed).to.equal(
+          expectedRDPCData[donorId].alignmentsFailed
+        );
+        expect(test_ca_hits[0]._source.alignmentsRunning).to.equal(
+          expectedRDPCData[donorId].alignmentsRunning
+        );
+        expect(test_ca_hits[0]._source.sangerVcsCompleted).to.equal(
+          expectedRDPCData[donorId].sangerVcsCompleted
+        );
+        expect(test_ca_hits[0]._source.sangerVcsFailed).to.equal(
+          expectedRDPCData[donorId].sangerVcsFailed
+        );
+        expect(test_ca_hits[0]._source.sangerVcsRunning).to.equal(
+          expectedRDPCData[donorId].sangerVcsRunning
+        );
+      });
```

Comment:
this should probably be done with a `Promise.all` or a `for await` cause `forEach` wouldn't know to await the async operation, so it might not reach these tests